CONTRIBUTING.md

```diff
@@ -420,6 +427,29 @@ $ ./node ./test/parallel/test-stream2-transform.js
 Remember to recompile with `make -j4` in between test runs if you change code in
 the `lib` or `src` directories.
 
+##### Test Coverage
+
+It's good practice to try to improve test coverage, as you add new features
+or work on an existing subsystems.
+
+To run the test suite with test coverage:
+
+```text
+$ make coverage
+```
+
+A detailed coverage report will be output to `coverage/index.html`.
```

Comment:
By the way, if you want C++ coverage too, I think you need `./configure --coverage` as well, and then (?) you’ll get `coverage/cxxcoverage.html` as well – that might not be necessary but maybe we should mention it?